### DISCUSSION TITLE
[Feature] Add "Ignore" button to duplicate detection.

### DISCUSSION THREAD

**Discussion by DanTulovsky** [immich-app/immich#10174] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, please consider adding an "Ignore" button when doing duplication detection. I realize I can unselect the deleted photos to achieve the same result, but that is a lot of needless clicking.

<img width="433" alt="image" src="https://github.com/immich-app/immich/assets/509887/58876719-5bff-4ef6-bb93-332ab8cb3c19">

Thank you
Dan

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by flesser** [comment]
https://github.com/immich-app/immich/pull/10189 adds a similar functionality
